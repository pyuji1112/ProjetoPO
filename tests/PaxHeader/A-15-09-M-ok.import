30 mtime=1635723209.950159637
