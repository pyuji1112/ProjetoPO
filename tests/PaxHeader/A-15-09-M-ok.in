30 mtime=1635724350.327969923
