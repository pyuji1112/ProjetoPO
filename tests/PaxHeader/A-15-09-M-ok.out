30 mtime=1635724334.496941343
